test/unit/slot/polyfill.js

```diff
@@ -1,5 +1,5 @@
 import '../../../src/index';
-import create from '../lib/create';
+import create from '../../lib/create';
```

Comment:
Not exactly, it was broken for some time, don't know how did it worked until now.
When I changed to the new build system it's just revealed.